src/main/java/org/broadinstitute/hellbender/tools/walkers/groundtruth/GroundTruthScorer.java

```diff
@@ -502,7 +503,7 @@ public void closeTool() {
         // write reports
         if ( reportFilePath != null ) {
             final GATKReport report = new GATKReport(
-                    BooleanAccumulator.newReportTable(qualReport, "qual", fbargs.probabilityRatioThreshold, omitZerosFromReport),
+                    BooleanAccumulator.newReportTable(qualReport, "qual", DEFAULT_RATIO_THRESHOLD, omitZerosFromReport),
```

Comment:
I think it is not. It is a FlowBaseRead update. right @ilyasoifer ?

Reply:
Yes and no, there were some issues in the calculation because of how the flow matrix was parsed in the haplotypeCaller (in the context of the haplotypeCaller we always allow pretty high error probability) that caused artefacts in the SNVQ calculation. 
The update made the SNVQ calculation more precise